src/main/java/com/aws/greengrass/clientdevices/auth/session/handlers/SessionCreationHandler.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.clientdevices.auth.session.handlers;
+
+import com.aws.greengrass.clientdevices.auth.api.DomainEvents;
+import com.aws.greengrass.clientdevices.auth.metrics.ClientDeviceAuthMetrics;
+import com.aws.greengrass.clientdevices.auth.session.events.SessionCreationEvent;
+
+import java.util.function.Consumer;
+import javax.inject.Inject;
+
+public class SessionCreationHandler implements Consumer<SessionCreationEvent> {
```

Comment:
Consider renaming this handler to reflect that its an _event handler_ and not Session-Creation handler (we actually have another class for that `SessionCreator`). 

Same goes for other _event_ handlers:
- `VerifyClientDeviceIdentityHandler`
- `CertificateSubscriptionHandler`